l10n_ch_fds_postfinance/wizards/fds_files_import_from_fds_wizard.py

```diff
@@ -110,6 +110,8 @@ def import_button(self):
             # process the files (done by the childrens of this module)
             for file in fds_files_ids:
                 self.process_files(file)
+                # We save the file if correctly processed
+                self.env.cr.commit()
```

Comment:
It's usually a bad idea to commit the main cursor.

I see we have the same at line https://github.com/OCA/l10n-switzerland/pull/570/commits/ad7fc00914723b2aaf24cd586253cbbb2aa10e07#diff-8fcb44095362db5b558733ac48f92b9353dba18438173751467d5239e3e41d17L72

Maybe you are looking at the savepoint case.
https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#never-commit-the-transaction

Reply:
@ecino gentle ping about this last issue on this PR